train_ilm.py

```diff
@@ -429,9 +429,10 @@ def train(args):
     else:
       print('from pretrained checkpoint')
       if args.model_name == ilm.constants.RUGPT3XL_MODEL_NAME:
-        model = RuGPT3XL.from_pretrained(args.model_name, tokenizer=tokenizer)
+        model = RuGPT3XL.from_pretrained(args.model_name, tokenizer=ilm.tokenize_util._get_tokenizer_state(tokenizer))
```

Comment:
`tokenizer` is the enum referring to the tokenizer name, not the actual tokenizer itself